backend/src/ee/services/permission/permission-fns.ts

```diff
@@ -131,12 +131,12 @@ function validateOrgSSO(actorAuthMethod: ActorAuthMethod, isOrgSsoEnforced: TOrg
   }
 }
 
-const escapeHandlebarsMissingMetadata = (obj: Record<string, string>) => {
+const escapeHandlebarsMissingDict = (obj: Record<string, string>, key = "identity.metadata") => {
```

Comment:
nit: i think it would be better to not have a default for the key so that it always needs to be defined. This would increase readability